apps/acceptance/lib/view.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var Marionette = require('marionette-client');
+
+function View(client) {
+    this.client = client.scope({searchTimeout: 5000});
+    this.actions = new Marionette.Actions(this.client);
+}
+
+View.prototype = {
+    //Get the root element of the app and return it.
```

Comment:
I don't think this comment add a lot of value to the code below. I would remove it.